### PR TITLE
Update set-up-new-message-encryption-capabilities.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the steps as aadrm module is depricated and is replaced by AIP service

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9060

File: microsoft-365/compliance/set-up-new-message-encryption-capabilities.md [hunk 1]
```diff
@@ -102,10 +102,10 @@ You can verify that your Microsoft 365 tenant is properly configured to use Micr
 
    - The default template names may be different from those displayed above. See [Configuring and managing templates for Azure Information Protection](/azure/information-protection/configure-policy-templates) for more.
 
-4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and run the Test-IRMConfiguration cmdlet to verify that it passes.
+4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and run the Test-IRMConfiguration cmdlet to verify that it passes. Connect to the [AIPService module](https://docs.microsoft.com/en-us/powershell/module/aipservice/?view=azureipps) to run the below cmdlet.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/set-up-new-message-encryption-capabilities.md]:
```suggestion
4. If the test fails with an error message **Failed to acquire RMS templates**, execute the following commands and run the Test-IRMConfiguration cmdlet to verify that it passes. Connect to the [AIPService module](/powershell/module/aipservice/?view=azureipps) to run the cmdlet.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**